<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEventDialog</class>
 <widget class="QDialog" name="NewEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Event</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/icons/calendar icon.svg</normaloff>:/resource/icons/calendar icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="eventname_label">
         <property name="text">
          <string>Event Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="about_label">
         <property name="text">
          <string>About Event</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="eventname_input"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="about_input"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="sendmessage_checkbox">
         <property name="text">
          <string>Send Message</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="done_button">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_button">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
